do_mpc/model.py

```diff
@@ -317,6 +317,8 @@ def __init__(self, model_type=None, symvar_type='SX'):
         self.flags = {
             'setup': False
         }
+        # Discrete-list of inputs for MINLP
+        self.integer = []
```

Comment:
Please give a more descriptive name to this list. Please make comment more clear:

e.g. "List of booleans to mark inputs as integer (vs. continuous). Integer inputs yield a MINLP problem.

Reply:
I just see it is a list of the names (as we discussed). Make this clear in the comment as well.